api/TODO.adoc

```diff
@@ -0,0 +1,23 @@
+== API traversal
```

Comment:
@pruivo I've added this to discuss your comment regarding obtaining different variants for the same construct.